journal.py

```diff
@@ -150,25 +108,18 @@ def edit_entry(request):
     return {'entry': entry}
 
 
-UPDATE_ONE_ENTRY = """
-UPDATE entries SET (title, text) = (%s, %s) WHERE id=%s
-"""
-
-
 @view_config(route_name='update-dynamic', request_method='POST', renderer='json')
 def edit_entry_dynamic(request):
     if request.authenticated_userid:
         if request.method == 'POST':
             try:
-                # Get rid of 'entry='
-                db_id = request.params.get('id', -1)[6:]
-                title = request.params.get('title', None)
-                text = request.params.get('text', None)
-                request.db.cursor().execute(UPDATE_ONE_ENTRY, (title, text, db_id))
+                edited = Entry.update(request)
             except psycopg2.Error:
                 return HTTPInternalServerError
-            text_markdown = markdown.markdown(text, extensions=['codehilite(linenums=True)', 'fenced_code'])
-        return {'title': title, 'text': text, 'text_markdown': text_markdown}
+            text_markdown = markdown.markdown(edited.text, extensions=['codehilite(linenums=True)', 'fenced_code'])
+        return {'title': edited.title,
+                'text': edited.text,
+                'text_markdown': text_markdown}
```

Comment:
Again, these value could (and probably should) be the result of calling some method on the Entry object, instead of sitting here.  It keeps your code more DRY to do so.